CONTRIBUTING.md

```diff
@@ -22,16 +22,35 @@ Struct names following Go conventions, e.g., `subjectCommonNameNotFromSAN`. Exam
 This will generate a new lint in the `lints/rfc` directory with the necessary
 fields filled out.
 
-**Choosing a Lint Result Level.** When choosing what `lints.LintStatus` your new
-lint should return (e.g. `Notice`,`Warn`, `Error`, or `Fatal`) the following
-general guidance may help. `Error` should be used for clear violations of RFC/BR
-`MUST` or `MUST NOT` requirements and include strong citations. `Warn` should be
-used for violations of RFC/BR `SHOULD` or `SHOULD NOT` requirements and again
-should include strong citations. `Notice` should be used for more general "FYI"
-statements that violate non-codified community standards or for cases where
-citations are unclear. Lastly `Fatal` should be used when there is an
-unresolvable error in `zlint`, `zcrypto` or some other part of the certificate
-processing.
+**Choosing Result Level.**  Lints return a single type of status:
+
+ * **Error:** `Error` can only be used for clear violations of `MUST` or `MUST
+   NOT` requirements and must include a specific citation.
+
+ * **Warning:** `Warn` can only be used for violations of `SHOULD` or `SHOULD
+   NOT` requirements and again should include strong citations. Many
+   certificate authorities block on both Error and Warning lints, and Warning
+   lints should not be used for non-deterministic errors.
```

Comment:
tiny nit: what's "non-deterministic" in this context? Without more background I might assume it's a lint that returns a warning finding some of the time but not others for the same input certificate. Do you mean more that this shouldn't be used for cases where the linter can't make a conclusive determination (e.g. due to missing information)? If so it might be better to be more explicit about that vs saying non-deterministic. WDYT?

Reply:
e.g., SN entropy

Reply:
I've added the example.

Reply:
Thanks, I think the example is enough to clarify :thumbsup: